docs/index.asciidoc

```diff
@@ -132,6 +138,25 @@ Configuration OK
 [2017-10-13T13:44:29,887][INFO ][logstash.runner          ] Using config.test_and_exit mode. Config Validation Result: OK. Exiting Logstash
 ----
 
```

Comment:
```suggestion
[discrete]
```

Reply:
I think an alternate way to fix is to make the header 4th-level like its peers instead of 3rd-level.

Reply:
Can you confirm (or show me how to)?

Reply:
It appears to be rendering correctly as is.